custom-build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2018 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+MODE=$1
+HOOKS_FILE=$2
+
+if [[ -n "${MODE}" ]]; then
+  case "${MODE}" in
+    asan)
+      export FUZZING_ENGINE=fsanitize_fuzzer
+      ;;
+    ubsan)
+      export FUZZING_ENGINE=fsanitize_fuzzer
+      export CFLAGS="-O2 -fno-omit-frame-pointer -g -fsanitize=undefined,fuzzer-no-link"
```

Comment:
you do you add fsanitize=undefined here and 
do not add in asan case?

Reply:
By default all benchmarks build with ASan.  `CFLAGS` and `CXXFLAGS` are specified in common.sh for the default case.  Maybe it makes sense to explicitly set the flags here for better readability?